servers/dart_server.dart

```diff
@@ -6,16 +6,19 @@ const String _HOST = '0.0.0.0';
 const String _GREET = 'Hello World';
 
 _startServer(arg) async {
-  var server = await HttpServer.bind(_HOST, 9292, shared: true);
-  await for (HttpRequest request in server) {
-    request.response
+  print('Start server');
+  final server = await HttpServer.bind(_HOST, 9292, shared: true);
+  server.listen((HttpRequest request) {
+      request.response
       ..write(_GREET)
       ..close();
-  }
+  });
 }
 
-void main() {
-  for (int i = 0; i < Platform.numberOfProcessors; i++)
+void main(List<String> args) {
+  final threads = (args.length>0?int.parse(args[0]):Platform.numberOfProcessors);
```

Comment:
Passing number of threads as an arg is not a prerequisite, i simply tuned the code for current workstation. 

Using less/more threads than CPUs number degrades performance in this case:
* 4 threads = 39549.25 req/sec
* 12 threads = 43685.43 req/sec

Reply:
Please see NodeJS result. https://docs.google.com/spreadsheets/d/1XxacDR05qPyVnb66yRfrEr_b0P84e56xqv_AHSnj41o/edit?usp=sharing

Reply:
Indeed your workstation has 2 CPUs and 4 hyper-threads, faster and more recent than mine 4 CPUs and 8 hyper-threads: thats' why spawning 8 processes bring no benefits. This is also why i use number of CPUs instead of fixed numbers.
In my case i get better results by using a number of processes equal to available hyper-threads (yours case too).